proposed/clock-meta.md

```diff
@@ -0,0 +1,154 @@
+# PHPDoc Meta Document
+
+## 1. Summary
+
+The purpose of using the ClockInterface is to provide a standard way to access the system 
+time, that would allow interopability when testing code that relies on the current time 
+rather than relying installing PHP extensions or use hacks like re-declaring the time() 
+function in other namespaces. 
+
+## 2. Why Bother?
+
+There are currently a few libraries that do provide the functionality on packagist, however 
+there is no interopability between these different libraries, as they ship with their own 
+clock interfaces. Symfony has a TimeMock library which uses namespace hacks to override the 
```

Comment:
Note: Symfony does not have a TimeMock library. Symfony provide a package named `symfony/phpunit-bridge` with a `Symfony\Bridge\PhpUnit\ClockMock` class:

> The `Symfony\Bridge\PhpUnit\ClockMock` class provided by this bridge allows you to mock the PHP’s built-in time functions `time()`, `microtime()`, `sleep()`, `usleep()` and `gmdate()`. Additionally the function `date()` is mocked so it uses the mocked time if no timestamp is specified.
>
>Other functions with an optional timestamp parameter that defaults to `time()` will still use the system time instead of the mocked time. This means that you may need to change some code in your tests. For example, instead of new `DateTime()`, you should use `DateTime::createFromFormat('U', time())` to use the mocked `time()` function.

For reference, see 

- https://symfony.com/doc/current/components/phpunit_bridge.html#clock-mocking
- https://github.com/symfony/phpunit-bridge/blob/v5.2.4/ClockMock.php

Reply:
I have updated this to be more accurate